roles/api_load/tasks/main.yml

```diff
@@ -1,20 +1,8 @@
 ---
-- block:
+- set_fact:
+    resources:
+    - "{{ role_path }}/templates/api_load.yml"
```

Comment:
Do we want to have the resources declared in the `tasks` rather than in `defaults`?